pkgs/applications/version-management/gitmux/default.nix

```diff
@@ -3,30 +3,28 @@
   buildGoModule,
   lib,
   testers,
-  gitmux,
   git,
 }:
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "gitmux";
   version = "0.11.2";
 
   src = fetchFromGitHub {
     owner = "arl";
-    repo = pname;
-    rev = "v${version}";
+    repo = "gitmux";
+    rev = "v${finalAttrs.version}";
     sha256 = "sha256-0Cw98hTg8qPu7BUTBDEgFBOpoCxstPW9HeNXQUUjgGA=";
   };
 
   vendorHash = "sha256-PHY020MIuLlC1LqNGyBJRNd7J+SzoHbNMPAil7CKP/M=";
 
   nativeCheckInputs = [ git ];
-  doCheck = true;
```

Comment:
Why was it removed?

Reply:
The tests are run by default.

https://github.com/NixOS/nixpkgs/blob/24.11/doc/languages-frameworks/go.section.md#skipping-tests-ssec-skip-go-tests

> buildGoModule runs tests by default.

Reply:
Cool, TIL.